complete/src/main/java/price/checker/service/AVRequestService.java

```diff
@@ -0,0 +1,59 @@
+package price.checker.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.client.RestTemplate;
+import price.checker.domain.StockPrice;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class is used to query to Alpha Vantage for stock data
+ */
+@RequiredArgsConstructor
+@Slf4j
+@Service
+public class AVRequestService {
+
+    private final String API_KEY = "KB7DSCK48G00WIRE";

```

Comment:
I mentioned this the other day but try using @Value with things like this (constants that could change at a later point) and adding the string to your properties file